src/Microsoft.Azure.WebJobs.Host/Bindings/ConverterManager.cs

```diff
@@ -2,46 +2,89 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System;
-using System.Collections;
 using System.Collections.Generic;
-using System.Reflection;
+using System.IO;
+using System.Linq;
 using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.Host.Config;
 using Newtonsoft.Json.Linq;
 
 namespace Microsoft.Azure.WebJobs
-{   
+{
     // Concrete implementation of IConverterManager
     internal class ConverterManager : IConverterManager
```

Comment:
>ConverterManager [](start = 19, length = 16)

This file is a near complete rewrite. Basically, changes in this file are:
- use the same means for OpenTypes and Exact Types (since exact type is just a subset of open type) 
- remove special casing. 
- the OpenType helpers got moved to that type.